ocaml/xapi/helpers.ml

```diff
@@ -1322,9 +1301,9 @@ end = struct
     | unknown    -> D.error "Stunnel.is_enabled: expected 'enabled' or 'disabled', but got: %s" unknown; false
 
   let restart ~__context ~accept =
+    (* generating certificates is handled by gencert systemd service *)
```

Comment:
I think this comment will be more confusing than helpful for future readers

Reply:
i've made it more clear now